m3/m3_common.py

```diff
@@ -329,7 +329,8 @@ def __init__(self, args_list = None):
             self._parse_args(args_list)
             self.ice = ICE()
             self.callback_q = Queue.Queue()
-            self.install_handler()
+            #Messes up mbus_snooper, why was this here?
+            #self.install_handler()
```

Comment:
yeah.. so, reading back, this was Pat trying to do too much with classes and inheritance, and I convinced myself that it was a good idea for there to be one "install handler" function (which, we'll note is one bloody line) that every "type" of "programmer" (goc / ein / mbus) would want to listen to their messages, so you have stuff like this:

```python
   class ein_programmer(object):
    TITLE = "EIN Programmer"
    DESCRIPTION = "Tool to program M3 chips using the EIN protocol."
    MSG_TYPE = 'b+'
```

Which will rely on the parent object to call install_handler during init but read the magic value of `MSG_TYPE` from `self` which will in this case change to `b+`....

A long-winded way of saying that `install_handler` was a bad idea and can/should be deleted